ports/nrf/boards/microbit_v2/mpconfigboard.mk

```diff
@@ -7,7 +7,7 @@ INTERNAL_FLASH_FILESYSTEM = 1
 
 CIRCUITPY_ALARM = 0
 CIRCUITPY_AUDIOMP3 = 0
-CIRCUITPY_BINASCII = 0
+CIRCUITPY_BINASCII = 1
```

Comment:
The default is that it is on, because of `CIRCUITPY_FULL_BUILD`, so we don't need to mark it on explicitly.
 ```suggestion
```

Reply:
Ah -- change made and tested